src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -0,0 +1,129 @@
+import React, {useContext, useState} from 'react';
```

Comment:
The code formatting can use some love in this file, probably time to add `prettier` to the project :P

Reply:
Yes! Let's add it! :D